test/sequence/boost_pfr.cpp

```diff
@@ -0,0 +1,202 @@
+/*=============================================================================
+    Copyright (c) 2021-2022 Denis Mikhailov
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+==============================================================================*/
+
+#include <boost/detail/lightweight_test.hpp>
+#include <boost/fusion/adapted/boost_pfr.hpp>
```

Comment:
Make a more complex test with a lot of ways to include pfr adapter.
For example, `boost/fusion/adapted/boost_pfr.hpp` here can be replaced to 'boost/fusion/adapted.hpp' , `boost/fusion/include/boost_pfr.hpp`, etc..